Lib/test/test_importlib/test_compatibilty_files.py

```diff
@@ -102,5 +102,5 @@ def test_spec_path_joinpath(self):
         self.assertIsInstance(self.files / 'a', CompatibilityFiles.OrphanPath)
 
 
-if __name__ == '__main__':
-    unittest.main()
+# This module is not suitable to be executed directly.
+# Use appropriate test tool.
```

Comment:
This copy/paste comment feels like a lot of extra noise. I believe it's the case that all Python modules have this behavior (they're only capable of being executed directly if the block is present). After all, there's lots of guidance we could provide, such as which test runners might be available to run it. I'd rather see a README that indicates that some (maybe which) modules are not executable. Or maybe move the test modules into subpackages and provide a README there that indicates that modules in that package are not executable. By adding this comment, you're adding an implied requirement that any new modules introduced need to include this comment as well, creating a boilerplate for certain modules.

I think I'd recommend (in my order of preference):

(a) Just leave it to the reader to know that modules without the block are not executable,
(b) Put some documentation in the developer guide that explains that some modules (specifically those without the block) are intentionally not executable, or
(c) Reorganize the tests such that they're in separate packages and document those packages.

One of those approaches would be minimally invasive and avoid boilerplate.

I do recognize the problem - how to capture the attention of the next person who comes along suspecting that the lack of executability was an oversight. From my perspective, the cruft of the comment is a greater cost than the value it provides.

WDYT?

Reply:
> From my perspective, the cruft of the comment is a greater cost than the value it provides.

Yeah, after seing the diff - I tend to agree. I think that `(b)` is the way to go.
I will open an issue about it and CC you there.

Thanks for the feedback! 👍

Reply:
SGTM. Thanks!